recipes/lynx/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+case "$(uname)" in
```

Comment:
I tend to do these for the `$(uname)` checks because more often than not I'll need windows/linux specific stuff down the line, and that way it all looks the same.